packages/loaders/openapi/tests/example_api.test.ts

```diff
@@ -2047,7 +2053,7 @@ describe('example_api', () => {
 
       const query = /* GraphQL */ `
         {
-          getSnack(snack_type: "soda")
+          getSnack(snack_type: soda, snack_size: large)
```

Comment:
@ardatan  Why did you add snack_size here?
I intentionally didn't add it, so this test both validates that input args override headers
AND that headers make input args nullable :)

Reply:
Oh sorry! Adding it back!